eahub/localgroups/urls.py

```diff
@@ -6,6 +6,7 @@
 urlpatterns = [
     urls.path("new", views.LocalGroupCreateView.as_view(), name="localgroups_create"),
     urls.path('<slug:slug>/claim', views.claim_group, name='claim_group'),
+    urls.path('<slug:slug>/report_inactive', views.report_group_inactive, name='report_group_inactive'),
```

Comment:
Can we change this to `report-inactive`? Dashes are generally more common in URLs.